src/pg/sql/41_observatory_augmentation.sql

```diff
@@ -0,0 +1,995 @@
+  poverty_quantile NUMERIC,
+  median_income_quantile NUMERIC,
+  gini_index_quantile NUMERIC,
+  income_per_capita_quantile NUMERIC,
+  housing_units_quantile NUMERIC,
+  vacant_housing_units_quantile NUMERIC,
+  vacant_housing_units_for_rent_quantile NUMERIC,
+  vacant_housing_units_for_sale_quantile NUMERIC,
+  median_rent_quantile NUMERIC,
+  percent_income_spent_on_rent_quantile NUMERIC,
+  owner_occupied_housing_units_quantile NUMERIC,
+  million_dollar_housing_units_quantile NUMERIC,
+  mortgaged_housing_unit_quantile NUMERIC
+
+) AS $$
+DECLARE
+  target_cols Numeric[];
+BEGIN
+target_cols := Array[
+                'total_pop_quantile',
+                'female_pop_quantile',
+                'male_pop_quantile',
+                'median_age_quantile',
+                'white_pop_quantile',
+                'black_pop_quantile',
+                'asian_pop_quantile',
+                'hispanic_pop_quantile',
+                'not_us_citizen_pop_quantile',
+                'workers_16_and_over_quantile',
+                'commuters_by_car_truck_van_quantile',
+                'commuters_by_public_transportation_quantile',
+                'commuters_by_bus_quantile',
+                'commuters_by_subway_or_elevated_quantile',
+                'walked_to_work_quantile',
+                'worked_at_home_quantile',
+                'children_quantile',
+                'households_quantile',
+                'population_3_years_over_quantile',
+                'in_school_quantile',
+                'in_grades_1_to_4_quantile',
+                'in_grades_5_to_8_quantile',
+                'in_grades_9_to_12_quantile',
+                'in_undergrad_college_quantile',
+                'pop_25_years_over_quantile',
+                'high_school_diploma_quantile',
+                'bachelors_degree_quantile',
+                'masters_degree_quantile',
+                'pop_5_years_over_quantile',
+                'speak_only_english_at_home_quantile',
+                'speak_spanish_at_home_quantile',
+                'pop_determined_poverty_status_quantile',
+                'poverty_quantile',
+                'median_income_quantile',
+                'gini_index_quantile',
+                'income_per_capita_quantile',
+                'housing_units_quantile',
+                'vacant_housing_units_quantile',
+                'vacant_housing_units_for_rent_quantile',
+                'vacant_housing_units_for_sale_quantile',
+                'median_rent_quantile',
+                'percent_income_spent_on_rent_quantile',
+                'owner_occupied_housing_units_quantile',
+                'million_dollar_housing_units_quantile',
+                'mortgaged_housing_unit_quantile'
+               ];
+
+END $$ LANGUAGE plpgsql  ;
+
+CREATE OR REPLACE FUNCTION OBS_GetDemographicSnapshot(geom geometry)
+RETURNS TABLE(
+  total_pop NUMERIC,
+  male_pop NUMERIC,
+  female_pop NUMERIC,
+  median_age NUMERIC,
```

Comment:
We considered doing this internally and concluded it was not worth it immediately -- @stuartlynn can speak to this.